src/main/java/uk/gov/hmcts/reform/pcqbackend/utils/ConversionUtil.java

```diff
@@ -228,4 +250,12 @@ public static String validateRequestHeader(List<String> requestHeaders) throws I
 
     }
 
+    private static PcqAnswerResponse createPcqRecord(ProtectedCharacteristics pcqDbRecord) {
```

Comment:
Could this method name be more specific? I.e. createConsolidationServicePcqRecord? Not that it matters, it's a private method so can be changed later if more return types are required.

Reply:
Done